src/i18n/index.ts

```diff
@@ -43,11 +45,13 @@ i18n.registerKeyset(Lang.En, KEYSET_NAME, {
     [Keyset.AllTags]: 'All topics',
     [Keyset.ActionSavedOnly]: 'Saved',
     [Keyset.AllServices]: 'All Services',
+    [Keyset.PromptSignInOnLike]: 'Please sign in to save your bookmarks',
```

Comment:
We don't need this keysets in package. Each app will stored these inside. It's flexible to customize

Reply:
please see comment above regarding UX - the message needs to be consistent across apps.

Reply:
I saw and answered))